mindjunkies/forums/views.py

```
@@ -58,43 +62,42 @@ def handle_topic_submission(self, request, course, course_slug):
-        
+
         # If form is not valid, return with errors and topics
         forum_topics = ForumTopic.objects.filter(course=course)
         return self.render_to_response(self.get_context_data(
             course=course,
             forum_topics=forum_topics,
             form=form
         ))
-    
+
     def handle_reply_submission(self, request, course_slug):
```

Ruff (0.8.2)

Missing return type annotation for public function `handle_reply_submission`

(ANN201)